tutor/src/components/assignment-links.cjsx

```diff
@@ -13,9 +13,10 @@ AssignmentLinks = React.createClass
   bindStore: ReferenceBookStore
 
   componentWillMount: ->
-    {courseId} = Router.currentParams()
+    {courseId} = _.defaults(@props, Router.currentParams() )
```

Comment:
For some reason, on my local, `@props` seems immutable, and the result of `_.defaults(@props, Router.currentParams() )` does not pull in the `courseId` from `Router.currentParams()` when i go directly to http://localhost:3001/course/1/assignment-links.

```
{courseId} = _.defaults(Router.currentParams(), @props)
```
appears to work.  Also, `Router.currentParams()` returns a deep clone each time, so using `default` on that should have no unexpected side-effects.